PyDSS/pyControllers/pyControllerAbstract.py

```diff
@@ -1,9 +1,19 @@
+import abc
 
-class ControllerAbstract():
-    def __init__(self):
 
+class ControllerAbstract(abc.ABC):
+
+    def __init__(self, controlledObj, Settings, dssInstance, ElmObjectList, dssSolver):
         """Abstract class CONSTRUCTOR."""
 
     def Update(self,  Priority, Time, UpdateResults):
         pass
 
+    def Name(self):
```

Comment:
Should all these methods have the decorator `abc.abstractmethod`?

Reply:
Yes they should. Made the required changes